README.md

```diff
@@ -339,98 +322,31 @@ public class Template
     public Int32 version;
 }
 ```
+Otherwise the client will raise a `Notify.Exceptions.NotifyClientException`.
+
+|`error.status_code`|`error.message`|
+|:---|:---|
+|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM of 10 requests per 20 seconds"`<br>`}]`|
+|`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (50) for today"`<br>`}]`|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Cannot send letters with a team api key"`<br>`]}`|
+|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Cannot send letters when service is in trial mode"`<br>`}]`|
+|`400`|`[{`<br>`"error": "ValidationError",`<br>`"message": "personalisation address_line_1 is a required property"`<br>`}]`|
 
-Otherwise the client will raise a `Notify.Exceptions.NotifyClientException`:
-<table>
-<thead>
-<tr>
-<th>
-
-`error["status_code"]`</th>
-<th>
-
-`error["message"]`</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>
-<pre>429</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "RateLimitError",
-    "message": "Exceeded rate limit for key type live of 10 requests per 20 seconds"
-}]
-</pre>
-</td>
-</tr>
-<tr>
-<td>
-<pre>429</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "TooManyRequestsError",
-    "message": "Exceeded send limits (50) for today"
-}]
-</pre>
-</td>
-</tr>
-<tr>
-<td>
-<pre>400</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "BadRequestError",
-    "message": "Cannot send letters with a team api key"
-]}
-</pre>
-</td>
-</tr>
-<tr>
-<td>
-<pre>400</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "BadRequestError",
-    "message": "Cannot send letters when service is in trial mode"
-}]
-</pre>
-</td>
-</tr>
-<tr>
-<td>
-<pre>400</pre>
-</td>
-<td>
-<pre>
-[{
-    "error": "ValidationError",
-    "message": "personalisation address_line_1 is a required property"
-}]
-</pre>
-</td>
-</tr>
-</tbody>
-</table>
 </details>
 
 
-### Arguments
+#### Arguments
 
+<details>
+<summary>
+Click here to expand for more information.
+</summary>
 
-#### `templateId`
+##### `templateId`
 
 Find by clicking **API info** for the template you want to send.
 
-#### `personalisation`
+##### `personalisation`
 	
 If a template has placeholders you need to provide their values. For example:
```

Comment:
should be
Dictionary<String, dynamic> personalisation = new Dictionary<String, dynamic>
{
    { "address_line_1", "23 Foo Road" },  # required
    { "address_line_2", "Bar Town" }, # required
    { "address_line_3", "London" },
    { "postcode", "BAX S1P" } # required
      ... # any other optional address lines, or personalisation fields found in your template
};